src/Honeycomb.OpenTelemetry/EnvironmentOptions.cs

```diff
@@ -36,5 +48,9 @@ private string GetEnvironmentVariable(string key, string defaultValue = "")
 
             return defaultValue;
         }
+
+        internal static string getErrorMessage(string humanKey, string key) {
+            return ($"Missing {humanKey}. Specify {key} environment variable, or the associated property in appsettings.json or the command line");
+        }

```

Comment:
```suggestion
        internal static string getErrorMessage(string humanKey, string key)
        {
            return $"Missing {humanKey}. Specify {key} environment variable, or the associated property in appsettings.json or the command line";
        }

```

Java braces 🙂 

Also I recommend PascalCase for the name